boards/arm/nrf52810_pca10040/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# The nrf52810_pca10040 board mirrors the nRF52832 DK hardware, which
+# needs to be considered by certain system initialization functionality
+# residing in nRFx.
```

Comment:
We used to write nrfx as nrfx. I think we should stay with this convention.
Anyway, I think it would be better to refer directly to the "system_nrf52810.c" file, instead of pointing to nrfx or MDK that resides inside it.